daemon/graphdriver/windows/windows.go

```diff
@@ -804,17 +804,17 @@ func writeLayer(layerData io.Reader, home string, id string, parentLayerPaths ..
 		return 0, err
 	}
 
-	size, err := writeLayerFromTar(layerData, w, filepath.Join(home, id))
-	if err != nil {
-		return 0, err
-	}
-
-	err = w.Close()
-	if err != nil {
-		return 0, err
-	}
+	defer func() {
+		if err2 := w.Close(); err2 != nil {
```

Comment:
Does it make sense to rename `err2` to `err` since that no longer conflicts with the return name?

Reply:
Yep. Done.